carte_cli/loader/carte_loader.py

```diff
@@ -64,12 +73,21 @@ def load_table(self, record: Union[DatabuilderTableMetadata, TableMetadata]):
                 raise ValueError(f"{e}\nFile name: {full_file_name}")
 
         frontmatter.dump(full_file_name, *extractor_metadata.to_frontmatter())
+        with open(
+            os.path.join(self.base_directory, self.tables_path, MANIFESTS_FILE), "a"
+        ) as manifests_file:
+            manifests_file.write(f"{record.get_file_name()}\n")
```

Comment:
We just write a list of the datasets present in the current extraction into the manifests file.